.github/workflows/release-pr.yml

```diff
@@ -0,0 +1,72 @@
+name: Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version to release"
+        required: true
+        default: "0.0.0"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  release:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Branch
+        id: branch
+        run: |
+          version=${{ github.event.inputs.version }}
+          version=${version%.*}
+          echo "branch=release-$version" >> $GITHUB_OUTPUT
+
+      - name: Checkout Repository
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+        with:
+          ref: ${{ steps.branch.outputs.branch }}
+
+      - name: Vars
+        id: vars
+        run: |
+          echo "current_version=$(yq '.version' <deploy/helm-chart/Chart.yaml)" >> $GITHUB_OUTPUT
+
+      - name: Find and Replace
+        uses: jacobtomlinson/gha-find-replace@a51bbcd94d000df9ca0fcb54ec8be69aad8374b0 # v3.0.2
+        with:
+          find: ${{ steps.vars.outputs.current_version }}
+          replace: ${{ inputs.version }}
+          include: "site/content/**/*.md"
+          regex: false
+
+      - name: Find and Replace
+        uses: jacobtomlinson/gha-find-replace@a51bbcd94d000df9ca0fcb54ec8be69aad8374b0 # v3.0.2
+        with:
+          find: "edge"
+          replace: ${{ inputs.version }}
+          include: "deploy/helm-chart/*.yaml"
```

Comment:
imagePullPolicy also needs to be updated to IfNotPresent in the values file.

https://github.com/nginxinc/nginx-gateway-fabric/pull/1378 should be a good reference for a past release in terms of all the updates that need to be made (there are some other irrelevant updates in there).

https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/release-process.md will also need to be updated to remove any unneeded steps in favor of using this workflow

Reply:
Updated the release process and values file